### DISCUSSION TITLE
Cant close

### DISCUSSION THREAD

**Discussion by Rifl33** [open-webui/open-webui#8033] [Q&A]
even after closing docker and closing container id localhost:3000 is always opened by it and cant close it. i did this docker run -d -p 3000:8080 -v ollama:/root/.ollama -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:ollama if this helps

**Comment by silentoplayz** [comment]
https://medium.com/@avishek2020/understanding-docker-restart-policies-restart-no-and-restart-yes-c1f8e217f983
